web/src/compose_closed_ui.ts

```diff
@@ -106,21 +118,23 @@ function toggle_direct_message_button_visibility(is_direct_message_narrow) {
     }
 }
 
-function update_buttons(text_stream, is_direct_message_narrow, disable_reply) {
+function update_buttons(
+    text_stream: string,
+    is_direct_message_narrow?: boolean,
+    disable_reply?: boolean,
+): void {
```

Comment:
Probably a future types cleanup  would just have this function accept an object, to have fewer optional parameters floating around.